scripts/tools/AddLanguageSupportForDelivery.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2021 (original work) Open Assessment Technologies SA;
+ *
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoDeliveryRdf\scripts\tools;
+
+use Exception;
+use oat\generis\model\data\import\RdfImporter;
+use oat\oatbox\extension\script\ScriptAction;
+use oat\oatbox\reporting\Report;
+use oat\oatbox\service\ServiceManager;
+
+/**
+ * Usages:
+ *
+ * php index.php "oat\taoDeliveryRdf\scripts\tools\AddLanguageSupportForDelivery"
+ */
+class AddLanguageSupportForDelivery extends ScriptAction
+{
+    private const APP_ROOT_PATH = __DIR__ . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR;
```

Comment:
it's already defined as ROOT_PATH in config/generis.conf.php:

`define('ROOT_PATH', '/var/www/html/');
`

Reply:
Suggestion applied. :)